primitives/src/locktime/absolute.rs

```diff
@@ -19,7 +19,7 @@ use crate::absolute;
 
 #[rustfmt::skip]                // Keep public re-exports separate.
 #[doc(inline)]
-pub use units::locktime::absolute::*;
+pub use units::locktime::absolute::{ConversionError, Height, ParseHeightError, ParseTimeError, Time};
```

Comment:
```suggestion
pub use units::locktime::absolute::{LOCK_TIME_THRESHOLD, ConversionError, Height, ParseHeightError, ParseTimeError, Time};
```

There is an `` error: unresolved link to `LOCK_TIME_THRESHOLD` ``  building the rustdocs.

Reply:
Thanks man!